arch/arc/core/fault_s.S

```diff
@@ -39,7 +39,11 @@ GTEXT(z_irq_do_offload);
 #endif
 
 /* the necessary stack size for exception handling */
-#define EXCEPTION_STACK_SIZE 384
+#define EXCEPTION_STACK_SIZE 768
```

Comment:
why don't we make this a Kconfig, instead of hard-coding a value here?

Reply:
done